### ISSUE TITLE
GPU support

### ISSUE THREAD

**Issue by yanivg10** [erikerlandson/ray-ubi#5]
Does this image support ray on GPUs?

**Comment by erikerlandson** [comment]
Hi @yanivg10, these images weren't designed or tested with gpus, but gpu support is definitely on my roadmap.

**Comment by yanivg10** [comment]
Thanks. What about Kubernetes? Can I use this image to run ray on k8s?

**Comment by erikerlandson** [comment]
@yanivg10 yes, these images should run on k8s. I would expect the operator deployment here to work on k8s:
https://github.com/erikerlandson/ray-odh-demo/tree/main/deploy/operator

(note, the corresponding open data hub deployment would not, as ODH is designed to run on openshift)

**Comment by yanivg10** [comment]
Thanks. Do I need the ray-operator-ubi image for k8s deployment?

**Comment by erikerlandson** [comment]
I don't think it is strictly necessary, but highly recommend using the ray operator for creating and managing ray clusters.  Run the ray operator pod, and then to create clusters you just create `RayCluster` CRD object, and the operator will use that to generate your cluster.  Ray is designed so that it expects cluster creation, scaling and tear-down to be managed by its operator.
